.gitignore

```diff
@@ -367,6 +367,3 @@ hs_err_pid*
 
 # The target folder contains the output of building
 **/target/**
-
-# F# vs code 
-.ionide/
```

Comment:
why did you remove this ?

Reply:
Oops this was a git merging error, I'll revert this.